locales/en.json

```diff
@@ -184,5 +184,6 @@
 	"Enable live count of notes": "Enable live count of notes",
 	"Enable smart table editor": "Enable smart table editor",
 	"Snippet Default Language": "Snippet Default Language",
-  "New notes are tagged with the filtering tags": "New notes are tagged with the filtering tags"
+  	"New notes are tagged with the filtering tags": "New notes are tagged with the filtering tags",
```

Comment:
Can you fix the indentation here?

Reply:
i suspect, that folder setted sort type.
please check new folder or "window.localStorage.clear()"

i think the undefined problem is caused by my mistake commit version